packages/core/components/FileList/index.tsx

```diff
@@ -135,8 +158,69 @@ export default function FileList(props: FileListProps) {
                                 innerRef(instance);
                             }
                         };
+                        const callbackRefGrid = (instance: FixedSizeGrid | null) => {
+                            gridRef.current = instance;
+                            if (isFunction(innerRef)) {
+                                innerRef(instance);
+                            }
+                        };
+                        // Custom onItemsRendered for grids
+                        // The built-in onItemsRendered from InfiniteLoader only supports lists
+                        const onGridItemsRendered = (gridData: any) => {
+                            const {
+                                visibleRowStartIndex,
+                                visibleRowStopIndex,
+                                visibleColumnStopIndex,
+                                overscanRowStartIndex,
+                                overscanRowStopIndex,
+                                overscanColumnStopIndex,
+                            } = gridData;
+
+                            // Convert injected grid props to InfiniteLoader list props
+                            const visibleStartIndex =
+                                visibleRowStartIndex * (visibleColumnStopIndex + 1);
```

Comment:
Start and Stop index values should be the same?

Reply:
The `visibleColumnStopIndex` yes but not the `visibleStartIndex` and `visibleStopIndex` which are differentiated by `visibleRowStartIndex` and `visibleRowStopIndex` respectively

Also, FWIW this is not a change from this PR but from Anya's work that added a thumbnail view.

Reply:
Ah, yes, my bad.  Thanks!